frontend/components/Header/Nav/EditionDropdown.js

```diff
@@ -32,8 +32,8 @@ export default () => {
     ];
 
     return (
-        <EditionDropdown>
+        <div className={editionDropdown}>
```

Comment:
Our previous use of `styled()` was causing problems for the React dev tools, but now that we're not- we should probably not `export default () => <JSX goes here/>` as this shows up as Unknown. `const Name = ()=> <JSX/>; export default const;` will show us Name in the tool.